torrent/no_torrent.go

```diff
@@ -0,0 +1,39 @@
+// +build !containers_image_include_torrent
```

Comment:
Why such a long name?   Why not just  !torrent?

Reply:
updated to `torrent`

Reply:
Because c/image is a library, and the tags share the namespace all over the resulting binary, incl. the 100MB ones OpenShift creates. I’d much prefer namespacing the tags for that reason.

(Stand-alone apps like podman do not have to worry about this, and can live with short tags.)